worker/state/statetracker.go

```diff
@@ -16,46 +16,48 @@ var ErrStateClosed = errors.New("state closed")
 // StateTracker describes a type which wraps and manages the lifetime
 // of a *state.State.
```

Comment:
StatePool
Maybe rename the interface as well?

Reply:
reworded, "state.State and state.StatePool"